MVPSt_2023_Feb_Competition/MVPSt_2023_Feb_Competition/Pages/HomePage.cs

```diff
@@ -0,0 +1,100 @@
+﻿using AventStack.ExtentReports;
+using MVPSt_2023_Feb_Competition.Utilities;
+using OpenQA.Selenium;
+//using SeleniumExtras.PageObjects;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static MVPSt_2023_Feb_Competition.Utilities.CommonDriver;
+
+namespace MVPSt_2023_Feb_Competition.Pages
+{
+    public class HomePage
+    {
+        // page objects
+        private IWebElement managelistingsButton => driver.FindElement(By.XPath(managelistingsButtonXP));
+        private IWebElement shareskillButton => driver.FindElement(By.XPath(shareskillButtonXP));
+
+        // Miscellaneous objects
+        private IWebElement signoutButton => driver.FindElement(By.XPath(signoutButtonXP));
+        private IWebElement signinLink => driver.FindElement(By.XPath(signinLinkXP));
+
+        // find the login button
+        //[FindsBy(How = How.XPath, Using = "/html/body/div[2]/div/div/div[1]/div/div[4]/button")]
```

Comment:
remember to remove all unwanted comments

Reply:
Removed unwanted comments from HomePage.cs